phase06/ConsoleApp/SearchEngine/IOWorks.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace SearchEngine;
+
+public class IOWorks
+{
+    public String[] GetInput()
```

Comment:
Same as what was said for EFCore reviews, our IO is not limited to command line interface or console. its better to use some interfaces
**+** splitting input violates SRP.